src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,392 @@
+                livenessProbe:
+                  failureThreshold: 6
+                  httpGet:
+                    path: /healthz
+                    port: http
+                    scheme: HTTP
+                  initialDelaySeconds: 120
+                  periodSeconds: 10
+                  successThreshold: 1
+                  timeoutSeconds: 5
+                name: kube-state-metrics
+                ports:
+                - containerPort: 8080
+                  name: http
+                  protocol: TCP
+                readinessProbe:
+                  failureThreshold: 6
+                  httpGet:
+                    path: /
+                    port: http
+                    scheme: HTTP
+                  initialDelaySeconds: 30
+                  periodSeconds: 10
+                  successThreshold: 1
+                  timeoutSeconds: 5
+                resources: {}
+                terminationMessagePath: /dev/termination-log
+                terminationMessagePolicy: File
+              dnsPolicy: ClusterFirst
+              restartPolicy: Always
+              schedulerName: default-scheduler
+              securityContext:
+                fsGroup: 1001
+                runAsUser: 1001
+              serviceAccount: ksm-kube-state-metrics
+              serviceAccountName: ksm-kube-state-metrics
+              terminationGracePeriodSeconds: 30
+              volumes:
+              - emptyDir: {}
+                name: tmpfs-data
+              - emptyDir: {}
+                name: tmpfs-user-data
+              - emptyDir: {}
+                name: tmpfs-tmp
+              - emptyDir: {}
+                name: tmpfs-cache
+        ```
+  
+  </Collapser>
+</CollapserGroup>
+
+3. Create a `ClusterRoleBinding`, or add to a previously created one the `ServiceAccount` of the application that is going to be monitored. All the workloads may share the same `ClusterRoleBinding`, but the `ServiceAccount` of each one must be added to it.
+4. Create a secret containing the New Relic license key. Each namespace needs its own secret.
+
+#### Configuration [#config-manual]
+
+When adding the manifest of the sidecar agent manually, you can use any agent configuration option to configure the agent behavior.
+**NEED A LINK HERE**
+
+#### Update to the latest version
+
+To update any of the components, you just need to modify the deployed yaml. 
+
+Updating any of the fields of the injected container will cause the pod to be re-created.
+
+<Callout variant="important">
+The agent cannot hot load the New Relic license key. After updating the secret, you need to rollout the deployments again.
+</Callout>
+
+#### Uninstall the Fargate integration
+
+To remove the injected container and the related resources, you just have  to remove the following:
+* The sidecar from the workloads that should be no longer monitored 
+* All the secrets containing the newrelic license 
+* `ClusterRole` and `ClusterRoleBinding` objects.
```

Comment:
Dots at the end of the bullet-points are not consistent.

Reply:
@invidian, thanks for noticing this! I'll add periods at the end of each bullet.